cheroot/connections.py

```diff
@@ -133,10 +137,9 @@ def get_conn(self, server_socket):
         ss_fileno = server_socket.fileno()
         socket_dict[ss_fileno] = server_socket
         try:
-            rlist, _, _ = select.select(list(socket_dict), [], [], 0.1)
-            # No available socket.
-            if not rlist:
-                return None
+            for sock in socket_dict:
+                self._selector.register(sock, selectors.EVENT_READ)
+            key_events = self._selector.select(timeout=0.1)
```

Comment:
would it be easier if we converted this into a dict right here?
something like
```suggestion
            key_events = {key.fd: event for key, event in self._selector.select(timeout=0.1)}
```

and this would allow to try/exept poping `ss_fileno` below and turning the rest of it into `rlist = list(key_events.keys())` (making the control flow nice and flat)

Reply:
oh, wait. does the order matter? then maybe use an ordered dict?

Reply:
Also, I noticed just now that `event` is not used anywhere. So it may make sense to just use a list and drop it. Like
```suggestion
            key_events = [key.fd for key, _event self._selector.select(timeout=0.1)]
```
WDYT?